wiki/advantages-and-features.md

```diff
@@ -0,0 +1,185 @@
+not having unexpected changes and having a way to voice your opinion without 
+needing to fork and split the community.
+
+
+### Comparison to CureCoin
+---
+#### What Is CureCoin?
+CureCoin is a cryptocurrency that aims to reward folding@home work
+
+
+#### Why Gridcoin Is Different
+
+* CureCoin only rewards folding@home --- Gridcoin rewards many projects   
+    * CureCoin is designed only for folding@home
+        * Only one project can be rewarded
+        * Expansion requires rework of the system
+
+    * Gridcoin is designed for BOINC projects
+        * Rewards many existing projects and allows for rewarding newly created BOINC projects
+        * More rewarded areas of study and more scientists running and starting projects. 
+
+^
+^
+
+* CureCoin involves Proof of Work ---  Gridcoin uses Proof of Stake
+    * More energy used in Proof of Work
+        * Less used energy on scientific work in CureCoin
+    
+    * With Gridcoin more energy goes directly to science
+        * Less energy used by Proof of Stake
+
+^
+^
+
+* Gridcoin has more active development and community
+    * Comparison of development activity in the last month:
+        * [Gridcoin's here](https://github.com/gridcoin-community/Gridcoin-Research/pulse/monthly) 
+        * [CureCoin's here](https://github.com/cygnusxi/CurecoinSource/pulse/monthly)
+ 
+
+### Comparison to OByte 
+---
+#### What Is OByte?
+OByte (formally Byteball) is a cryptocurrency that focuses on various different
+aspects such as smart contracts. Among other bots, it has a bot that distributes out 
+Bytes for work on [World Community Grid](https://www.worldcommunitygrid.org)
+  
+(note you can earn both Gridcoin and Bytes at the same time)
+
+
+#### Why Gridcoin Is Different
+
+* OByte's main focus is different from Gridcoin 
+    * Science aspect is a secondary or tertiary goal
+        * Fewer OBytes sent to reward science and more goes to other distribution methods
+    
+    * For Gridcoin science is a primary goal
+        * Larger percentage rewards science 
+
+^
+^
+
+* OByte's bots are heavily centralized 
+    * Happening through one bot that could stop at any point
+        * Funding to this bot comes entirely through a grant from a fund 
+          controlled currently by 4 people
+
+    * In Gridcoin, coins are distributed as new coins in the network not through a single fund or bot
+
+^
+^
+
+* OByte only rewards World Community Grid and no other BOINC project
+    * On any other project, you can't earn OBytes for it
+
+    * With Gridcoin there are {{ site.data.whitelist.projects.size }} projects you can contribute to
+
+^
+^
+
+
+### Comparison to Primecoin
+---
+#### What Is Primecoin?
+Primecoin is a crytpocurrency that uses a proof of work algorithm to 
+find [Cunningham chains](https://en.wikipedia.org/wiki/Cunningham_chain) of prime numbers.
+
+
+#### Why Gridcoin Is Different
+* Primecoin can only find prime numbers and do no scientific computation 
```

Comment:
```suggestion
* Primecoin can only find prime numbers and no other scientific computations
```

Reply:
I think I'm going to leave "do" in there to help clarify, but I will add "other" in there. I think both "computation" or "computations" are correct, oddly enough. I don't really know why that is. English grammar is weird.